loader/packaging/requirements.json

```
@@ -0,0 +1,76 @@
+{
+    "version": 1,
+    "host": {
+        "linux": {
+            "x64": {
+                "supported": true,
+                "glibc": {
+                    "supported": true,
+                    "description": "From centOS 7",
+                    "min": "2.17.0"
+                },
+                "musl": {
+                    "supported": true,
+                    "description": "From alpine 3.19"
```

In which case, should it be this?
```suggestion
                    "description": "From alpine 3.19",
                    "min": "1.2.4"
```

---

Currently, the injector does not detect the musl version, so enforcing a minimum version results in a failure

---

Well, that's problematic :eyetwitch: